packages/teams-js/src/public/pages.ts

```diff
@@ -442,6 +442,8 @@ export namespace pages {
       const saveEvent = new SaveEventImpl(result);
```

Comment:
Please add unit tests that will test this new code. The unit tests are in pages.spec.ts, which uses a file called utils.ts to mock the post messaging between the current window and the child window. We should have tests that verify the new behavior (message sent to child window if there is a child window and no one in the current window has registered a handler)

Reply:
I have added unit tests now.